engine/runtime/src/main/java/org/enso/interpreter/caches/Cache.java

```diff
@@ -303,6 +300,32 @@ private T loadCacheFrom(TruffleFile cacheRoot, EnsoContext context, TruffleLogge
     }
   }
 
+  private boolean isBlobDigestValid(File path, TruffleLogger logger, ByteBuffer blobBytes, M meta) {
+    if (!needsDataDigestVerification) {
+      return true;
+    }
+    LOGGER.info("checking blob digest is valid: " + path);
+    if (Boolean.getBoolean("enso.blob.valid")) {
```

Comment:
It would be nice to document those magic parameters somewhere. And we can group them under i.e. `enso. debug` (similar to `--deug.dev-tools` etc. ide flags)